tests/Organization.py

```diff
@@ -321,13 +340,17 @@ def testCreateRepositoryWithAutoInit(self):
         self.assertEqual(
             repo.url, "https://api.github.com/repos/BeaverSoftware/TestPyGithub"
         )
+        self.assertTrue(repo.has_pages)
+        self.assertTrue(repo.has_wiki)
 
     def testCreateFork(self):
         pygithub = self.g.get_user("jacquev6").get_repo("PyGithub")
         repo = self.org.create_fork(pygithub)
         self.assertEqual(
             repo.url, "https://api.github.com/repos/BeaverSoftware/PyGithub"
         )
+        self.assertFalse(repo.has_wiki, False)
+        self.assertFalse(repo.has_pages, False)
```

Comment:
How is this not a syntax error, wow. Anyway, remove the unneeded second argument.